### PR TITLE
updated_documemtation

### PR DESCRIPTION
**Pull request by ajkhattak**
The PR revises the documentation to make it more user-friendly and easy to follow and also updates the outdated files.

## Additions
- None

## Removals
- None

## Changes
- No changes to the code

## Testing
1. All existing tests passed

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Passes all existing automated tests
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

File: realizations/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Model build/run
+
```
Comment (GreyEvenson-NOAA) [inline realizations/README.md:2]: Maybe add text to say "Example realization files are provided in this directory."

File: realizations/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+### Build
+To build LASAM (see [instructions](https://github.com/NOAA-OWP/LGAR-C/blob/ajk/doc_update/INSTALL.md#build-1))
```
Comment (GreyEvenson-NOAA) [inline realizations/README.md:4]: "To build LASAM for use in NextGen"

File: realizations/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+### Run
+To run LASAM (see [instructions](https://github.com/NOAA-OWP/LGAR-C/blob/ajk/doc_update/INSTALL.md#run-1))
```
Comment (GreyEvenson-NOAA) [inline realizations/README.md:7]: "To run LASAM via NextGen:

File: configs/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+## Configuration File
+Example configuration files are provided here. To build and run the given examples see the instructions ([here](https://github.com/NOAA-OWP/LGAR-C/blob/ajk/doc_update/INSTALL.md#build-standalone-mode)).
```
Comment (GreyEvenson-NOAA) [inline configs/README.md:2]: "Example configuration files are provided in this directory"? I was half-expecting the work 'here' to point to a link but realized that it refers to this directory.

**Review by GreyEvenson-NOAA — APPROVED** [review]
Looks good!

Using this documentation, I was able to build/run in standalone using using Phillisburg and Bushland example data, the unit tests, and using the synthetic data. I was also able to build/run for NextGen using all of the example realization files (for LASAM and LASAM coupled with SFT and SMP).

I included minor suggested changes to README files.

File: configs/README.md (hunk 4 above)
Comment (ajkhattak) [reply configs/README.md:2]: good point, I revised it.

**Pull request merged**